<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title> RNA Analysis software list </title>
</head>
<body>
    <p>
       * RNAcommender v0.1-beta.1 – Genome-wide Recommendation of RNA-protein Interaction<br><br>
* PcircRNA_finder – circRNA Prediction in Plants<br><br>
* Metaxa2 2.1.3 – Identification and Classification of Small Subunit (SSU) rRNA Sequences<br><br>
* MirCompare – Algorithm to Compare Plant and Mammalian miRNAs<br><br>
* cisASE – Detecting Putative Cis-regulated Allele-specific Expression in RNA Sequencing data<br><br>
* MIMEAnTo – Profiling functional RNA in Mutational Interference Mapping Experiments<br><br>
* 16S classifier – Fast and Accurate Taxonomic Classification of 16S rRNA Sequences<br><br>
* mirPRo 1.0.1 – Tool for miRNA-seq Analysis<br><br>
* IRESPred – Prediction of Cellular and Viral Internal Ribosome Entry Site (IRES)<br><br>
* RNAgraphdist 1.0 – Calculation of Graph-distance Distribution of the Boltzmann ensemble of RNA Secondary Structures<br><br>
* GORAP 2 – Genomewide ncRNA Annotation Pipeline<br><br>
* LRIscan – Predict Conserved, Genome-wide Long range RNA-RNA Interactions<br><br>
* miRNAFold – Ab initio miRNA Precursor Search in Genomes<br><br>
* TarPmiR – microRNA Target Site Prediction<br><br>
* spongeScan – Detecting microRNA Binding Elements in lncRNA Sequences<br><br>
* JAR3D – Predicting RNA 3D Motifs in Sequences<br><br>
* SePIA – RNA Sequence Processing, Integration, and Analysis<br><br>
* UEA sRNA Workbench 4.1 ALPHA – Analysing micro RNA and other small RNA data from High-Throughput Sequencing devices<br><br>
* RNArobo 2.1.0 – RNA Structural Motif Search tool<br><br>
* rMAPS v1.0.1 – RNA Map analysis and Plotting server for alternative Exon Regulation<br><br>
* TSCAN 1.7.0 – Pseudo-time Reconstruction and Evaluation in Single-cell RNA-seq Analysis<br><br>
* DeAnnIso – Detection and Annotation of IsomiR from small RNA sequencing data<br><br>
* tRF2Cancer 1.0 – A Web Server to Detect tRFs and their expression in multiple Cancers<br><br>
* Trnascan-SE 2.0 – Search for tRNA Genes in Genomic Sequence<br><br>
* RDDpred 1.0 – Random Forest RDD Classifier<br><br>
* Ballgown – Isoform-level Differential Expression Analysis in R<br><br>
* StringTie 1.2.2 – Transcript Assembly and Quantification for RNA-Seq<br><br>
* TopHat 2.1.1 – Short Read Aligner for RNA-Seq Experiments<br><br>
* RME 1.1 – RNA Secondary Structure Prediction incorporating Experimental Constraints<br><br>
* RNAex – RNA Secondary Structure Prediction Server<br><br>
* RNA22 v2 – microRNA Target Detection<br><br>
* MINTbase – Exploration of Mitochondrial and Nuclear tRNA Fragments<br><br>
* ChemiRs – Web Server for microRNAs and Chemicals<br><br>
* miRlastic 1.0 – Integrative Analysis of miRNA and mRNA Expression data<br><br>
* miRQuest – Integration of tools on a Web Server for microRNA Research<br><br>
* isomiR-SEA 1.60 – RNA-Seq analysis tool<br><br>
* miRNA Digger – Genome-wide novel miRNA Mining<br><br>
* miR-isomiRExp – Analysis of Expression of miRNA at the miRNA/isomiR levels<br><br>
* IntSplice – Prediction of the Splicing Consequences of Intronic Single-nucleotide Variations in the Human Genome<br><br>
* MatureBayes – Web-based tool for the Computational Identification of Mature miRNAs<br><br>
* MiRduplexSVM – MiRNA-Duplex Prediction and Evaluation Methodology<br><br>
* tRNAmodpred – Predicting Posttranscriptional Modifications in tRNAs<br><br>
* MetaDiff – Differential Expression Analysis on RNA-Seq<br><br>
* MISIS 2.7 – Mapped short Interfering RNA Spots Identification Software<br><br>
* MIRUMIR – microRNA as Biomarker In Cancer<br><br>
* SpliceJumper – Splicing Junction Calling from RNA-Seq data<br><br>
* SSC 0.1 – Sequence Scan for CRISPR<br><br>
* LncDisease 1.41 – Predicting lncRNA-disease Associations<br><br>
* SRAMP – Sequence-based RNA Adenosine Methylation Site Predictor<br><br>
* m6Apred – Identification and Analysis of the N6-methyladenosine in Saccharomyces Cerevisiae Transcriptome<br><br>
* MethyRNA – Identificationof N6-methyladenosine Sites<br><br>
* SeqFold – Genome-scale Reconstruction of RNA Secondary Structure integrating Experimental Measurements<br><br>
* Stem Kernels 193 – Directed acyclic graph kernels for structural RNA analysis<br><br>
* DRME – Differential RNA Methylation Analysis<br><br>
* RED 0.0.3 – RNA Editing sites Detector<br><br>
* iKNN – Classifier for Non-coding RNA Identification in Organelle Genome level<br><br>
* EBChangePoint 0.1.0 – Empirical Bayes Change-point Model<br><br>
* EmpiricalGO – Measure Enrichment using an Empirical Sampling Approach<br><br>
* DAFS 0.0.3 – Simultaneous Aligning and Folding of RNA Sequences by Dual Decomposition<br><br>
* CentroidFold 0.0.15 – Predict RNA Secondary Structure from RNA Sequence<br><br>
* SplAdder – Detection and Quantification of Alternative Splicing Events from RNA-Seq data<br><br>
* GESS – Exon-skipping Scanner Detection program for RNA-seq data<br><br>
* RSEARCH 1.1 – Sequence Database Searching with RNA Structure Queries<br><br>
* PKNOTS 1.07 – RNA Pseudoknot Prediction<br><br>
* RNABOB 2.2 – Fast Pattern Searching for RNA Secondary Structures<br><br>
* QRNA 2.03d – Prototype Noncoding RNA Genefinder<br><br>
* CONUS 1.0 – Exploration of Lightweight Stochastic Context Free Grammars<br><br>
* pRNAm-PC – Predicting N6-methyladenosine sites in RNA sequences<br><br>
* ESA – Programs and data sets for RNA Alignment<br><br>
* miRTrail – Analysis of Relationships between miRNAs and mRNAs<br><br>
* novoMiRank – Prioritizing novel predicted miRNA Precursors for Experimental Validation<br><br>
* miFRame – Analysis and Visualization of miRNA sequencing data in Neurological Disorders<br><br>
* miRNACon – Influence of Confounding Factors Age and Sex on microRNA<br><br>
* iMiRNA-SSF – Identification of MicroRNA based on Structure and Sequence Features<br><br>
* OEFinder 0.0.1 – Identify Ordering Effect Genes in Single Cell RNA-seq data<br><br>
* Scotty – Power Analysis for RNA Seq Experiments<br><br>
* RNASeqPowerCalculator – Calculate the Power and Sample size for RNA-Seq Differential Expression<br><br>
* MirMark – Prediction of microRNA Targets both at Site-level and UTR-level<br><br>
* LocARNA 1.8.7 – Global and Local Alignment of RNA<br><br>
* OfftargetFinder – web tool for species-specific RNAi Design<br><br>
* FR3D 20120611 / WebFR3D – Find Structural Motifs in RNA 3D Structures<br><br>
* pirnaPre 1.0.0 – piRNA Targets on mRNAs in Mouse<br><br>
* DEsi – Design Engine of siRNA<br><br>
* siPRED – Predicting siRNA Efficacy using Various Characteristic Methods<br><br>
* rQuant 2.1 – Transcriptome Quantitation from RNA-seq Experiments<br><br>
* FuMa – Reporting Overlap in RNA-seq Detected Fusion Genes<br><br>
* SI-FI 3.2 – RNAi (RNA interference) Off-target Prediction<br><br>
* CLASS 2.1.2 – Constraint-based Local Assembly and Selection of Splice Variants<br><br>
* Rcorrector – Error Correction for Illumina RNA-seq Reads<br><br>
* TargetS – microRNA Target Prediction in Different Gene Regions<br><br>
* LncRNA-ID – Long non-coding RNA IDentification using Balanced Random Forest<br><br>
* miRMOD 0.3 – miRNA Modification Identification tool<br><br>
* miRA 1.1.1 – micro RNA Identification tool<br><br>
* miRge – Rational and Efficient Approach to miRNA-seq<br><br>
* SMiRK – an Automated Pipeline for miRNA Analysis<br><br>
* RNAentropy 1.0 – Compute RNA Secondary Structure Conformational Entropy<br><br>
* Transcriptator 2.0 – Annotate Transcripts and Assembled Reads from RNA-Seq data<br><br>
* CLRNASeq 1.00 – RNA-Seq Analysis<br><br>
* FlipFlop 1.8.0 – Fast Lasso-based Isoform Prediction as a Flow Problem<br><br>
* CapR 1.0.4 – Revealing Structural Specificities of RNA-binding protein Target Recognition<br><br>
* CDSfold 0.2 – Designing Protein-coding RNA Sequence<br><br>
* RNAstructure 5.7 – Prediction & Analysis of RNA Secondary Structure<br><br>
* WU-CRISPR – WashU gRNA Designer for CRISPR/Cas9 Knockout<br><br>
* deGPS 2.0 – Detecting Differential Expression in RNA-sequencing Studies<br><br>
* BRAKER1 1.6 – RNA-seq based Eukaryotic Genome Annotation pipeline<br><br>
* PLNseq 1.0 – Poisson Lognormal Distribution for High-throughput matched RNA-sequencing Read Count data<br><br>
* contamDE 1.0 – Differential Expression analysis of RNA-seq data for Contaminated Tumor Samples<br><br>
* CircTest 0.1.0 / DCC 0.3.2 – Identification and Quantification of Circular RNAs from Sequencing Data<br><br>
* PCS 1.5 – Identify and Analyze Conserved K-mers in Pairwise Alignment<br><br>
* NanoStringNorm 1.1.19 – Normalize NanoString miRNA and mRNA Data<br><br>
* meRanTK 1.1.0 – Methylated RNA Analysis ToolKit<br><br>
* SeqBuster 3.1 – small RNA analysis of deep Sequencing data<br><br>
* multiMiR 1.0.1 – Integration of microRNA-target Interactions along with their Disease and Drug Associations<br><br>
* Vienna RNA Package 2.1.9 – RNA Secondary Structure Prediction & Comparison<br><br>
* CAMUR – Classifier with Alternative and MUltiple Rule-based models<br><br>
* CracTools 1.22 / digitagCT 1.0.2 – Integrated pipelines for RNA-Seq Analysis using CRAC<br><br>
* CRAC 2.4.0 – An Integrated RNA-Seq Read Analysis<br><br>
* CPSS 1.1 – Computational Platform for analysis of Small RNA Deep Sequencing data<br><br>
* miRD – pre-microRNA Detection<br><br>
* SNAPR – Scalable Nucleotide Alignment Program RNA-seq<br><br>
* MREdictor – MicroRNA Recognition Elements Prediction Tool<br><br>
* RSF 0.1.0 – Automated Transcriptome-wide Reconstruction of RNA Secondary Structures<br><br>
* iMcRNA / miRNA-dis – microRNA Precursor Identification based on Distance Structure Status Pairs<br><br>
* miRNA-deKmer – Identification of microRNA precursor with the degenerate K-tuple or Kmer strategy<br><br>
* iRNA-Methyl – Identifying N6- methyladenosine sites using Pseudo Nucleotide Composition<br><br>
* DEGseq 1.24.0 – Differentially Expressed Gene Identification for RNA-seq data<br><br>
* sPARTA 1.12 – small RNA-PARE Target Analyzer<br><br>
* htSeqTools 1.16.0 – Quality Control, Visualization and Processing for High-Throughput Sequencing data<br><br>
* casper 2.4.0 – Characterization of Alternative Splicing based on Paired-End Reads<br><br>
* TSSi 1.16.0 – Transcription Start Site Identification<br><br>
* BASIS v1 – Bayesian Analysis of Splicing IsoformS<br><br>
* GPSeq 0.5 – Using the Generalized Poisson distribution to model sequence read counts from high throughput sequencing experiments<br><br>
* WemIQ – Isoform Quantification method for RNA-seq data<br><br>
* DEXUS 1.10.0 – Identifying Differential Expression in RNA-Seq Studies with Unknown Conditions<br><br>
* rPredictor – Prediction of Ribosomal RNA Secondary Structure<br><br>
* StarScan 1.2.0 – Scanning small RNA Targets from Degradome Sequencing data<br><br>
* RNAalishapes 2.4.7 – Secondary Structure Prediction using Shape Abstraction<br><br>
* pAliKiss 1.0.7 – Predict Kissing Hairpin Motifs from Aligned Sequences<br><br>
* KnotInFrame 2.0.8 – Prediction of -1 Ribosomal Frameshift Events<br><br>
* RNAshapes 3.3.0 – RNA Analysis Package based on Abstract Shapes<br><br>
* aCMs 1.2.1 – Stochastic Homology Search tools<br><br>
* rnaifold 2.0 – RNA Inverse Folding and Molecular Design<br><br>
* SETTER / MultiSETTER – RNA SEcondary sTructure-based TERtiary Structure Similarity Algorithm<br><br>
* shortran 0.43 – A pipeline for small RNA-seq data analysis<br><br>
* PPfold 3.1.1 – Predict Consensus Secondary Structure of RNA Alignments<br><br>
* nagnag 1.0.3 – Identifyng and Quantifying NAGNAG Alternative Splicing<br><br>
* tweeDEseq 1.16.0 – Analyzing RNAseq Count data<br><br>
* cellHTS2 2.35.0 / web cellHTS2 – Analysis of Cell-based RNAi Screens<br><br>
* DESeq2 1.10.0 – Differential Expression Analysis for Sequence Count Data<br><br>
* RNAither 2.18.0 – Statistical analysis of high-throughput RNAi screens<br><br>
* Tax4Fun 0.3.1 – Predicting Functional Profiles from Metagenomic 16S rRNA data<br><br>
* DSSR 1.4.1 – Dissecting the Spatial Structure of RNA<br><br>
* miRPD – miRNA Protein Disease Associations<br><br>
* RPASuite v0.02 – RNA Processing Analysis Suite<br><br>
* RNAcop 1.0.3 – Optimizing RNA structures by Sequence Extensions<br><br>
* ContextMap 2.5.0 – Context-based mapping of RNA-seq data<br><br>
* polyester 1.6.0 – Simulate RNA-seq Reads<br><br>
* PlantMiRNAPred – Classification of Real and Pseudo Plant Pre-miRNAs<br><br>
* MaturePred – Identification of microRNAs within novel Plant pre-miRNAs<br><br>
* RNAMotifScanX 0.0.5 – Graph Alignment approach for RNA Structural Motif Identification<br><br>
* CRISPRseek 1.10.0 – Design of Target-specific Guide RNAs in CRISPR-Cas9, Genome-editing Systems<br><br>
* Vfold 2.0 – RNA Structure and Folding Thermodynamics Prediction<br><br>
* EBSeq 1.11.0 / EBSeq-HMM 1.4.0 – RNA-Seq Differential Expression Analysis<br><br>
* dsCheck — Off-target Search website for dsRNA-mediated RNAi<br><br>
* PLEK 1.2 – Predictor of Long Non-coding RNAs and mRNAs based on K-mer Scheme<br><br>
* RIPSeeker 1.10.0 – de novo RIP Peak Predictions based on HMM.<br><br>
* ProMISe 1.8.0 – Infer miRNA-mRNA interactions using paired expression data from a single sample<br><br>
* Mirsynergy 1.7.0 – Detect Synergistic miRNA Regulatory Modules by Overlapping Neighbourhood Expansion<br><br>
* TFAM 1.4 – tRNA Function Classifier<br><br>
* Quantas 1.0.8 – Analyze alternative Splicing using RNA-Seq<br><br>
* CIMS 1.0.5 – Crosslinking Induced Mutation Site analysis<br><br>
* mCarts 1.2.0 – Hidden Markov model to predict Clustered RNA Motif sites<br><br>
* SimSeq 1.3.0 – Nonparametric Simulation of RNA-Seq Data<br><br>
* piPipes 1.4.12 – piRNA Pipeline Collection<br><br>
* GRM 0.2.1 – Single Cell RNA-seq De-noise Tools<br><br>
* Annocript 1.1.2 – Annotation of Transcriptomes<br><br>
* FASTR – Format for Concomitant Representation of RNA Sequence and Secondary Structure Information<br><br>
* TF2Lncrna – Identifying common Transcription Factors for a list of lncRNA Genes<br><br>
* IRcall 1.0.2 / IRclassifier 1.0.2 – Intron Retention Call / Classifier<br><br>
* metaRNASeq 1.0.2 – Meta-analysis of RNA-seq data<br><br>
* miRDeep2 2.0.0.7 / miRDeep* 3.6 – Discovering Known and Novel miRNAs from Deep Sequencing Data<br><br>
* miRPlant v4 – An Integrated Tool for Identification of Plant MiRNA from RNA Sequencing Data<br><br>
* RNASeqBrowser V2 – A Genome Browser for Strand Specific RNAseq Reads<br><br>
* CADBURE 20150810 – Evaluating Aligner performance on your RNA-Seq dataset<br><br>
* RNASeqMut 0.7 – RNA-Seq Mutation Detection<br><br>
* CHIP-RNA-SEQPRO 1.1 – ChIP-RNA-sequencing-processing<br><br>
* miRtest 1.8 – Combined miRNA- and mRNA-testing<br><br>
* miR-PREFeR 0.24 – microRNA PREdiction From small RNA-seq data<br><br>
* ProbRNA 20131213 – Modeling RNA Structure Probing data<br><br>
* RNASeqGUI 1.0.0 – GUI for the Identification of Differentially Expressed Genes<br><br>
* riboPicker 0.4.3 – Identification and Removal of rRNA-like Sequences.<br><br>
* TASR 1.1 – Annotate Transposable Elements using siRNAs Mapping<br><br>
* comPARE – PARE Validated miRNA Targets<br><br>
* miTRATA – microRNA Truncation and Tailing Analysis<br><br>
* RNAbrowse 1.4 – RNA-Seq de novo Assembly Results Browser<br><br>
* RSEM 1.2.22 – RNA-Seq Expression Estimation by Expectation-Maximization<br><br>
* FlaiMapper 1.2.0 – Fragment Location Annotation Identification Mapper<br><br>
* RNAfdl 1.1.2 – RNA Force Directed Layout<br><br>
* compcodeR 1.4.0 – Benchmarking Differential Expression Methods for RNA-seq data<br><br>
* ExomePeak 1.8.0 – Exome-based Analysis for RNA Epigenome Sequencing Data<br><br>

    </p>
</body>
</html>